gcb/presubmit.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+set -e
+
+REMOTE_INSTANCE_NAME="projects/google.com:raksha-ci/instances/raksha_rbe_instance"
+RESULT_UI="--bes_backend=buildeventservice.googleapis.com --bes_timeout=60s \
+           --bes_results_url=https://source.cloud.google.com/results/invocations/ \
+           --project_id=google.com:raksha-ci"
+BAZEL_TEST="bazelisk test --config=remote --remote_instance_name=${REMOTE_INSTANCE_NAME} ${RESULT_UI}"
+BAZEL_BUILD="bazelisk build --config=remote --remote_instance_name=${REMOTE_INSTANCE_NAME} ${RESULT_UI}"
+
+# Run all the tests.
+${BAZEL_TEST} //src/...
+# Verifies that the following targets build fine:
+#  - Arcs parser and proto works.
+#  - Arcs manifest tests that we do not yet handle parse correctly.
+#  - Authorization logic code with bazel.
+#  - Arcs manifest examples.
+${BAZEL_BUILD} //third_party/arcs/examples:consume third_party/arcs/proto:manifest_cc_proto \
+   //src/analysis/souffle/tests/arcs_manifest_tests_todo/... \
+   //rust/tools/authorization-logic:authorization_logic \
```

Comment:
Now that we are actually using the authorization logic in tests, this should probably be removed.

Actually, should we swap the order of the build and test invocations? In a sense, the build checks are "more basic" in that they just make sure that everything still builds without invoking any deep Raksha functionality. Therefore, we would expect these build commands to pass more frequently than the test commands, making them more cacheable.

Reply:
done.